security/stm32mp1/privateKey.pem

```diff
@@ -0,0 +1,7 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
This file is encrypted, what is the default password for it?

Reply:
Woops: `fio`